resources/views/bios/create.blade.php

```diff
@@ -0,0 +1,26 @@
+@extends('layout')
+
+@section('content')
+
+    <div class="container">
+        <div class="row">
+            <div class="col-md-6">
+                <h1>Create Bio</h1>
+
+                <ul class="errors">
+                    @foreach ($errors->all() as $message)
+                        <li>{{ $message }}</li>
+                    @endforeach
+                </ul>
+
+                {{ Form::open(array('action' => 'BiosController@store', 'class' => 'new-bio-form')) }}
```

Comment:
5.3 array can be 5.4 array OR EVEN 5.6 ARRAY IF THERE WAS SUCH A BEAUTIFUL THING.

Greenfield project bleeding edge PHP versions FTW.